.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "gcsio/googleapis"]
```

Comment:
How long to wait until we will have this dependency in Maven?

Reply:
I think that they expect it next week, if so maybe better to hold-on with merging until after their release?

Reply:
Actually, this is definition for proto-only dependency: https://github.com/googleapis/java-storage/pull/922/files#diff-bff76a2b2494c2d6fda58db2766038487d0bf19e5815ab248c480a1f7f19e793

So, I think that we should wait until it will be available in Maven. Not sure if it worth migrating to GAPIC though.

Reply:
Makes sense, will hold off till next week, for protos to be available in maven central. 

Could you please review the rest of the PR.

Reply:
Reviewed.

Note, that if we decide to go with submodule then we need to pin it somehow to specific commit/tag because right now it makes our builds non-deterministic